ios-target/View/Home/HomeViewController.swift

```diff
@@ -46,6 +46,8 @@ class HomeViewController: UIViewController {
     viewController.viewModel.topics = viewModel.topics
   }
   
+  @IBAction func prepareForUnwind(segue: UIStoryboardSegue) { }
```

Comment:
Do you need this action?

Reply:
Without this action it won't let me access the exit point on the storyboard to dismiss this view